tests/test_workspaces/test_views.py

```
@@ -16,6 +18,72 @@
+    mock_connector.return_value = mock_instance
+    mock_instance.connection.locations.count.return_value = 1
+    
+    response = api_client.get(url)
+
+    assert response.status_code == 200
+    assert response.data["message"] == "Netsuite connection is active"
+    
+    cache.set(cache_key, True, timeout=timedelta(hours=24).total_seconds())
+
+    # Assert cache is present and true
+    assert cache.get(cache_key) == True
```

Ruff (0.11.9)

Avoid equality comparisons to `True`; use `if cache.get(cache_key):` for truth checks

Replace with `cache.get(cache_key)`

(E712)